File: projects/front/onlineshopping/src/app/model/product.ts
import { Category } from './category';

export class Product{

public name:string;
public categoryId:number;
public price:number;

public image:string;
public email:string;
public cityId:number;
public description:string;
public ownerName:string;
public phone:string;
public register:Date;
public userId:number;
public pageCount:number;

    constructor(public  id:number){
        
    }
   

}

export class City{
    public name:string;
    
    constructor(public id:number){

    }
}

File: projects/back/shopping/src/main/java/az/developia/shopping/model/Product.java
package az.developia.shopping.model;

import java.sql.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name="product")
@Getter
@Setter
public class Product {
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
private Integer id;
private String name;
private Integer categoryId ;
private double price;
private boolean isNew;
private String  image;

private Integer cityId;
private String description;
private String ownerName;
private String phone;
private Date register;
private Integer userId;
private  Integer page;

}
